agent/app/agent_capability.go

```diff
@@ -77,10 +77,13 @@ const (
 	capabilityExecCertsRelativePath                        = "certs"
 	capabilityExternal                                     = "external"
 	capabilityServiceConnect                               = "service-connect-v1"
+	capabilityGuardDuty                                    = "guard-duty-patrol"
 
 	// network capabilities, going forward, please append "network." prefix to any new networking capability we introduce
 	networkCapabilityPrefix      = "network."
 	capabilityContainerPortRange = networkCapabilityPrefix + "container-port-range"
+	//using guardDuty.service as a temporary placeholder
+	guardDutyServiceFilePath = "/usr/lib/systemd/system/guardDuty.service"
```

Comment:
Q. Should we define this in the [config.go] instead of here since it is not a capability?

Reply:
I agree, moved this to config.go